.golangci.yml

```diff
@@ -142,3 +142,13 @@ issues:
     - linters:
         - lll
       path: "pkg/apis/submarinerconfig/v1alpha1/zz_generated.swagger_doc_generated.go"
+
+    # Ignore underscores in var names in auto-generated files
+    - linters:
+        - stylecheck
+      path: "pkg/apis/submarinerconfig/v1alpha1/zz_generated.swagger_doc_generated.go"
+
+    # This needs to be changed in the main submariner-io/submariner repo first
```

Comment:
This is defined in _pkg/apis/submarinerconfig/v1alpha1/types.go_ in this repo and not submariner. I don't think the Go struct is used outside of this repo so should be safe to change it and remove this exclusion.